lib/matplotlib/_mathtext.py

```diff
@@ -2336,7 +2338,8 @@ def _genfrac(self, ldelim, rdelim, rule, style, num, den):
     def genfrac(self, s, loc, toks):
         return self._genfrac(
             toks.get("ldelim", ""), toks.get("rdelim", ""),
-            toks["rulesize"], toks["style"],
+            toks["rulesize"], self._MathStyle(
+                toks.get("style", self._MathStyle.TEXTSTYLE.value)),
```

Comment:
I think that should be self._MathStyle.TEXTSTYLE (as the code downstream accesses the `.value` attribute itself)?

Reply:
Was just about to post a comment about the same thing. This should work, because the whole thing is wrapped inside `self._MathStyle()`. So, an `enum` variable is correctly passed.

However, this does two conversions (`enum` to `int`, then back to `enum`). Is there a more optimal way to default to text style?

Reply:
Ah sorry I missed that.  FWIW I would just turn the enum into an IntEnum and accept some confusion between ints and MathStyles; converting back and forth (while unlikely to cause a significant slowdown) doesn't seem to be really necessary or help legibility here.